packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
+const GLOB_PATTERNS = '**\/*.{html,js,css}';
```

Comment:
I'm not sure it's wise having a set of defaults in the plugin, either they are required or the defaults should be moved into sw-build.

What do you think?